pages/governance/slashing_a_validator.md

```diff
@@ -0,0 +1,44 @@
+# Slashing a validator
+
+The chain supports slashing of misbehaving validators through governance vote.
+
+## Proposal Message
+
+The proposal should contain a single [MsgSlashValiator](https://github.com/dydxprotocol/v4-chain/blob/protocol/v4.0.0/proto/dydxprotocol/govplus/tx.proto#L19) message for each validator that should be slashed. See the inline comments for details on each field.
+
+**💡 Important: Both `tokens_at_infraction_height` and `slash_factor` must be set correctly, otherwise undelegations and redelegations might be slashed disproportionately to the rest of the validator's stake.**
```

Comment:
Can we add explanations around how to pick these values? For example, how do you pick the right value for the height given the governance process takes a few days and etc